addons/product_expiry/migrations/13.0.1.0/openupgrade_analysis.txt

```diff
@@ -4,3 +4,5 @@ product_expiry / stock.production.lot     / product_expiry_reminded (boolean): N
 ---XML records in module 'product_expiry'---
 NEW ir.ui.view: product_expiry.view_production_lot_view_tree
 DEL ir.ui.view: product_expiry.view_move_form_expiry_simple
+# NOTHING TO DO: product_expiry.view_move_form_expiry_simple view will be
```

Comment:
Please don't do these changes on the original file, but copy it over `openupgrade_analysis_work.txt` and do the changes there.